ui/src/androidTest/java/com/xmartlabs/bigbang/ui/recyclerview/common/ListActivity.java

```diff
@@ -0,0 +1,27 @@
+package com.xmartlabs.bigbang.ui.recyclerview.common;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
+public abstract class ListActivity<T extends RecyclerView.Adapter> extends Activity {
+  RecyclerView recyclerView;
+
+  protected final T adapter = createAdapter();
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(com.xmartlabs.bigbang.ui.test.R.layout.activity_with_list);
+    setupRecyclerViewAdapter();
+  }
+
+  private void setupRecyclerViewAdapter() {
+    recyclerView = (RecyclerView) findViewById(com.xmartlabs.bigbang.ui.test.R.id.recycler_view);
```

Comment:
And why are you using a test package in there?
Shouldn't stuff here be more generic?